core/gather/gatherers/devtools-log.js

```diff
@@ -28,12 +30,56 @@ class DevtoolsLog extends FRGatherer {
 
     /** @param {LH.Protocol.RawEventMessage} e */
     this._onProtocolMessage = e => this._messageLog.record(e);
+
+    this._disableAsyncStacks = () => {};
+  }
+
+  /**
+   * Enables `Debugger` domain to receive async stacktrace information on network request initiators.
+   * This is critical for tracking attribution of tasks and performance simulation accuracy.
+   * @param {LH.Gatherer.FRProtocolSession} session
+   */
+  async _enableAsyncStacks(session) {
```

Comment:
Looks like this doesn't get enabled for legacy, though? If so, kind of worried we don't have any smoke test coverage of async stack trace 😬

Reply:
It does get called in legacy. `startSensitiveInstrumentation` is run in `beforePass` by default:

https://github.com/GoogleChrome/lighthouse/blob/4c5c65e256591ade1381872e7274e7f94ff33be2/core/gather/base-gatherer.js#L75-L83

Reply:
I don't think the DevtoolsLog gatherer is run in legacy though?

Reply:
Ah your right, we would need to do this in legacy driver :/

This is indeed concerning for lack of smoke tests

Reply:
I've added a smoke test for this. I tested and dbw now fails if async stacks aren't enabled.